scio-bigquery/src/main/scala/com/spotify/scio/bigquery/BigQueryIO.scala

```diff
@@ -199,7 +199,7 @@ final case class BigQueryTable(table: Table) extends BigQueryIO[TableRow] {
     params.insertErrorTransform(params.extendedErrorInfo.coll(data.context, wr))
 
     if (params.writeDisposition == WriteDisposition.WRITE_APPEND) {
-      throw new NotImplementedError("BigQuery future with append not implemented")
+      NotImplementedTap("BigQuery with append does not support tap")
```

Comment:
I kind of feel like this error message is somewhat conflicting with the class name `NotImplementedTap` ? To me `NotImplementedTap` implies that in a future version we might implement it, whereas `"does not support tap"` implies it's not supported by design, and never will be... what do you think?

Reply:
Yeah you might be right, renamed this.